src/zapv2/ajaxSpider.py

```diff
@@ -29,13 +29,13 @@ def status(self):
         """
         This component is optional and therefore the API will only work if it is installed
         """
-        return next(self.zap._request(self.zap.base + 'ajaxSpider/view/status/').itervalues())
+        return self.zap._request_value('ajaxSpider/view/status/')
```

Comment:
done! Can I leave `_request_value`? I may to think about to use it in that code generator you mentioned at irc.